### PR TITLE
Note that PKCE is not supported

### PR DESCRIPTION
**Pull request by hickford**
Most other OAuth servers support PKCE, so it's worthwhile to note this limitation of GitHub.  at https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/authorizing-oauth-apps

Also because of security implementations https://datatracker.ietf.org/doc/html/draft-ietf-oauth-security-topics#section-4.7.1

> Clients MUST ensure that the AS supports PKCE before using PKCE for CSRF protection. If an authorization server does not support PKCE, state or nonce MUST be used for CSRF protection.

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by steveward** [comment]
Thanks for the PR @hickford. This was brought up in https://github.com/github/docs/issues/22270, and we're still waiting for a technical review there. I'll get this triaged for review as well.

**Comment by skedwards88** [comment]
@github/ecosystem-apps-reviewers Can you review this PR from a technical perspective?

**Comment by Cruzillian** [comment]
Just wanted to let y'all know that this is me, JC! My new number is
(361)252-4428 Cruzlove baby! The Phantom Cruzillian!

**Comment by nikymorg** [comment]
Tagging in @hpsin to review for @github/ecosystem-apps-reviewers. 🙇

File: content/apps/oauth-apps/building-oauth-apps/authorizing-oauth-apps.md [hunk 1]
```diff
@@ -316,6 +316,10 @@ To build this link, you'll need your OAuth Apps `client_id` that you received fr
 
 {% endtip %}
 
+## Limitations
+
+There is no support for PKCE ([RFC 7636](https://datatracker.ietf.org/doc/html/rfc7636)), see [feature request](https://github.com/orgs/community/discussions/15752).
```
Comment (hpsin) [inline content/apps/oauth-apps/building-oauth-apps/authorizing-oauth-apps.md:321]: This is more appropriate around line 57, where a developer would be looking for the supported parameters.

**Review by hpsin — CHANGES_REQUESTED** [review]

File: content/apps/oauth-apps/building-oauth-apps/authorizing-oauth-apps.md (hunk 1 above)
Comment (hpsin) [reply content/apps/oauth-apps/building-oauth-apps/authorizing-oauth-apps.md:321]: Perhaps "The PKCE (Proof Key for Code Exchange ) parameters `code_challenge` and `code_challenge_method` are not supported at this time" and leave it at that. Linking to the feedback discussion isn't standard for our documentation.

Comment (hickford) [reply content/apps/oauth-apps/building-oauth-apps/authorizing-oauth-apps.md]: Done.

Comment (hpsin) [reply content/apps/oauth-apps/building-oauth-apps/authorizing-oauth-apps.md]: Thank you!

**Review by hpsin — APPROVED** [review]
Thanks for including this @hickford ! Looking forward to replacing it with details of how to use these once we are able to add it.

**Comment by hickford** [comment]
@hpsin thanks. Please could you also review related PR https://github.com/github/docs/pull/24964 ?

**Review by skedwards88 — APPROVED** [review]
Thanks for this addition @hickford and thanks for the review @hpsin ! We'll get this merged down.

**Pull request merged**